examples/wind/src/layers/delaunay-cover-layer/delaunay-cover-layer.js

```diff
@@ -92,9 +92,10 @@ export default class DelaunayCoverLayer extends Layer {
         setParameters(gl, {
           blend: true,
           blendFunc: [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA],
-          blendEquation: gl.FUNC_ADD
+          blendEquation: gl.FUNC_ADD,
+          depthTest: true,
```

Comment:
Yeah more over, this issue is also present in demo version except if you run field layer first, which is setting this state and other layers depend on it, which made the issue even  complicated as nothing came out when comparing particle and elevation layers in both versions.